scripts/generate-chart-versions.sh

```
@@ -0,0 +1,73 @@
+  # Fail loudly rather than silently guessing a version: publishing a
+  # chart under a made-up placeholder tag would be worse than failing
+  # the build, since it could get pushed to the registry unnoticed.
+  if ! tag=$(git -C "${path}" describe --tags --abbrev=0 --exclude '*-nightly*' 2>/dev/null); then
+    echo "ERROR: no real (non-nightly) release tag reachable from ${path} — refusing to guess a version" >&2
+    exit 1
+  fi
+  sha=$(git -C "${path}" rev-parse --short=7 HEAD 2>/dev/null || echo "unknown")
+  echo "${tag}|${sha}"
+}
+
+> chart-versions.txt
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)